server.py

```diff
@@ -228,3 +231,17 @@ def semesterIDType(reqstr):
 
   # Convert Officer objects back to dictionaries before returning
   return jsonify([x.__dict__ for x in officers])
+
+calurl = 'https://calendar.google.com/calendar/ical/ca149os3pmnh0dcopr1jn2negg%40group.calendar.google.com/public/basic.ics'
```

Comment:
Maybe consider moving this Global variable to the top of the file in like a globals/constant area at the top